packages/dashboards/src/lib/components/timeseries-widget/chart-presets/xy-chart/xy-chart.component.less

```diff
@@ -0,0 +1,48 @@
+@import (reference) "@nova-ui/bits/sdk/less/nui-framework-variables.less";
+@import (reference) "@nova-ui/bits/sdk/less/mixins.less";
+
+@nui-legend-deemphasized-opacity: 0.3;
+@nui-legend-series-content-margin: 3px;
+
+.legend-container {
+    .legend-description-container {
+        & .legend-description {
+            line-height: @nui-line-height-default;
+            .text-overflow(ellipsis);
+            padding-left: @nui-space-xs;
+            padding-right: @nui-space-xs;
+            min-width: 130px;
+
+            &-primary {
+                font-weight: @nui-font-weight-semibold;
+                &:not(.nui-text-link-small) { 
+                    .setCssVariable(color, nui-color-text-default);
+                }
+                &:not(:last-child) {
+                    margin-bottom: -@nui-legend-series-content-margin;
+                }
+            }
+
+            &-secondary {
+                &:not(.nui-text-link-small) { 
+                    .setCssVariable(color, nui-color-text-secondary);
+                }
+                &:not(:first-child) {
+                    margin-top: -1px;
+                }
+            }
+        }
+    }
+
+    &:not(:hover)&.state-hidden {
```

Comment:
this class doesn't look correct to me. here's what it looks like in the nui-legend-series less file:

https://github.com/solarwinds/nova/blob/0a504b1c5c55cafc5bd02e2845e6f0da1791f591/packages/charts/src/legend/legend-series/legend-series.component.less#L140

Reply:
If we use the `[seriesRenderState]` input on the nui-legend-series, I think we can use the classes that are automatically applied to the series instead of this custom class `state-hidden`.